config/routes.rb

```diff
@@ -9,6 +9,9 @@
   get 'sessions/create'
   get 'sessions/destroy'
 
+  get 'users/line_items', to: 'users#line_items'
+  get 'users/orders', to: 'users#orders'
```

Comment:
Can we write code that assign `controller` to both without repeating `users#`

Reply:
done